src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/hooks.ts

```diff
@@ -20,3 +25,45 @@ export const useGetFormActionErrors = () => {
     flatFormErrors,
   };
 };
+
+export const useHasEnoughMembersWithPermissions = ({
+  decisionMethod,
+  selectedAction,
+  createdIn,
+}: {
+  decisionMethod: DecisionMethod;
+  selectedAction: Action;
+  createdIn?: number;
+}) => {
+  const { watch } = useFormContext();
+  const formValues = watch();
+
+  const requiredRoles = getPermissionsNeededForAction(
+    selectedAction,
+    formValues,
+  );
+
+  const { thresholdPerRole } = useDomainThreshold({
+    domainId: createdIn,
+    requiredRoles,
+  });
+
+  const { countPerRole } = useEligibleSignees({
+    domainId: createdIn,
+    requiredRoles,
+  });
+
+  if (!thresholdPerRole || !requiredRoles) {
+    return true;
+  }
+
+  const hasEnoughMembersWithPermissions = requiredRoles.every(
+    (role) => countPerRole[role] >= thresholdPerRole[role],
+  );
+
+  const showNotEnoughMembersWithPermissionsNotification =
+    decisionMethod === DecisionMethod.MultiSig &&
+    !hasEnoughMembersWithPermissions;
+
+  return !showNotEnoughMembersWithPermissionsNotification;
```

Comment:
Oh boy, yeah I left this in a bit of a mess with those last few lines! Originally I made this entire hook as `useShowNotEnoughMembersWithPermissionsNotification` but later renamed it when I realised it might be useful elsewhere, but clearly forgot to tidy up this last bit!